### ISSUE TITLE
Can't set the maximum header depth

### ISSUE THREAD

**Issue by aslongaspossible** [hipstersmoothie/obsidian-plugin-toc#9]
Every time I set a new  maximum header depth to be x, it finally set the minimum header depth to x and set the maximum header depth to 6.

**Comment by Elijer** [comment]
Yeah I don't even know what's happening. It's so close to being really helpful, but it's unusable for me -- I want to only display "#", "##" and "###", but it acts unpredictable. Sometimes the depths I select get reflected in the TOC and other times they don't. Regardless, whenever I open the settings up again, the sliders are both set to their defaults again.

**Comment by bellaisgroovy** [comment]
I couldn't use it to generate header level 1 only
